cli-plugins/insight/cli-configuration.hbs.md

```diff
@@ -2,31 +2,15 @@
 
 This topic explains how to configure the Tanzu Insight plug-in.
 
-> **Note** All [required setup](cli-overview.md) must be completed in addition to configuring the CLI.
-
-
 ## <a id='set-tar-cert'></a>Set the target and certificate authority certificate
 
-Set the target endpoint and CA certificate by running:
+Note: These instructions are for the recommended configuration where ingress is enabled. For instructions on non-ingress setups, see [Using Encryption and Connection](../../scst-store/using-encryption-and-connection.hbs.md#additional-resources) for more details.
```

Comment:
Should we call see `[Using Encryption and Connection]` to `see [Using Encrypted Connection]`